src/crawlee/_request.py

```diff
@@ -99,7 +99,7 @@ def __len__(self) -> int:
 class BaseRequestData(BaseModel):
     """Data needed to create a new crawling request."""
 
-    model_config = ConfigDict(populate_by_name=True)
+    model_config = ConfigDict(populate_by_name=True, arbitrary_types_allowed=True)
```

Comment:
This is kinda scary. Maybe we could make `HttpHeaders` a `RootModel` instead (https://docs.pydantic.dev/dev/concepts/models/#rootmodel-and-custom-root-types)?

Reply:
Ok, should be solved...